README.md

```diff
@@ -12,3 +12,12 @@ HttpKernel adapter for use of Symfony and Laravel frameworks with PHP-PM. See ht
 
         composer require php-pm/httpkernel-adapter:dev-master
 
+> **Note**: Make sure your `AppKernel` is autoloaded in your `composer.json`:
```

Comment:
It's worth then to point out that it's not necessary in most cases or for symfony standard editions. Just to not confuse people